commit 4f06f3d1201d5630720f089da2bdeab9792ab36b
Author: Brad Smith <brasmith@redhat.com>
Date:   Wed Aug 24 16:03:28 2022 -0400

    add ENABLE_SYNTHETIC_DATA_REQUEST_HTTP_API to playbook task for convenience with ephemeral deployments

diff --git a/deployment/playbooks/roles/clowder/tasks/clowder.yml b/deployment/playbooks/roles/clowder/tasks/clowder.yml
index 59c97a80..e8cef4bc 100644
--- a/deployment/playbooks/roles/clowder/tasks/clowder.yml
+++ b/deployment/playbooks/roles/clowder/tasks/clowder.yml
@@ -31,6 +31,7 @@
         --set-parameter "postigrade/IMAGE_TAG={{ lookup('env', 'POSTIGRADE_IMAGE_TAG') | default('latest', True) }}"  \
         --set-parameter "cloudigrade/CLOUDIGRADE_ENVIRONMENT={{ env }}" \
         --set-parameter "cloudigrade/DJANGO_SETTINGS_MODULE=config.settings.prod" \
+        --set-parameter "cloudigrade/ENABLE_SYNTHETIC_DATA_REQUEST_HTTP_API={{ lookup('env', 'ENABLE_SYNTHETIC_DATA_REQUEST_HTTP_API') | default('False', True) }}" \
         --set-parameter "cloudigrade/AWS_ACCESS_KEY_ID={{ lookup('env', 'AWS_ACCESS_KEY_ID') | b64encode }}" \
         --set-parameter "cloudigrade/AWS_SECRET_ACCESS_KEY={{ lookup('env', 'AWS_SECRET_ACCESS_KEY') | b64encode }}" \
         --set-parameter "cloudigrade/CW_AWS_ACCESS_KEY_ID={{ lookup('env', 'AWS_ACCESS_KEY_ID') | b64encode }}" \
@@ -65,6 +66,7 @@
         --set-parameter "cloudigrade/IMAGE_TAG={{ (cloudigrade_deployment_host == 'local') | ternary(new_image_tag.stdout, 'latest') }}" \
         --set-parameter "cloudigrade/CLOUDIGRADE_ENVIRONMENT={{ env }}" \
         --set-parameter "cloudigrade/DJANGO_SETTINGS_MODULE=config.settings.prod" \
+        --set-parameter "cloudigrade/ENABLE_SYNTHETIC_DATA_REQUEST_HTTP_API={{ lookup('env', 'ENABLE_SYNTHETIC_DATA_REQUEST_HTTP_API') | default('False', True) }}" \
         --set-parameter "cloudigrade/AWS_ACCESS_KEY_ID={{ lookup('env', 'AWS_ACCESS_KEY_ID') | b64encode }}" \
         --set-parameter "cloudigrade/AWS_SECRET_ACCESS_KEY={{ lookup('env', 'AWS_SECRET_ACCESS_KEY') | b64encode }}" \
         --set-parameter "cloudigrade/CW_AWS_ACCESS_KEY_ID={{ lookup('env', 'AWS_ACCESS_KEY_ID') | b64encode }}" \
